### PR TITLE
docs: explain default-trackers param format in rpc-spec

### PR DESCRIPTION
**Pull request by trim21**

### CODE REVIEW

**Review by ckerr — CHANGES_REQUESTED** [review]

File: docs/rpc-spec.md [hunk 1]
```diff
@@ -527,7 +527,7 @@ Response arguments: `path`, `name`, and `id`, holding the torrent ID integer
 | `blocklist-url` | string | location of the blocklist to use for `blocklist-update`
 | `cache-size-mb` | number | maximum size of the disk cache (MB)
 | `config-dir` | string | location of transmission's configuration directory
-| `default-trackers` | list of default trackers to use on public torrents
+| `default-trackers` | list of default trackers to use on public torrents | a '\n' joined string, each line for a tracker
```
[the remark this answers is not in the document]
Comment (trim21) [reply docs/rpc-spec.md]: I'm not sure if `'\n'` is ok becuase I didn't test it, so I just use `\n`

**Review by ckerr — CHANGES_REQUESTED** [review]
This line LGTM. Thanks for the fast response!

Please update `trackerList` on line 161 to use the identical text, e.g. adding the BEP link

**Review by ckerr — APPROVED** [review]
Thanks again!

**Comment by ckerr** [comment]
Notes: Documentation improvements.

**Pull request merged**